Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM ubuntu:focal
+FROM ubuntu:noble
 
 # https://github.com/rust-transit/gtfs-to-geojson.git (rust app)
 FROM rust:latest as builder
 WORKDIR /
 # this repo has no tagged releases ; we pin the version based on latest verified commit instead
 RUN git clone https://github.com/rust-transit/gtfs-to-geojson.git
-RUN git -C gtfs-to-geojson checkout 9ca9a25b895ba1b2fdf4d04e92895afec52d0608
+RUN git -C gtfs-to-geojson checkout 604dd7a1f7d0f97f1a89c9b061a41e3094dfd310
```

Comment:
Diff: https://github.com/rust-transit/gtfs-to-geojson/compare/9ca9a25b895ba1b2fdf4d04e92895afec52d0608..604dd7a1f7d0f97f1a89c9b061a41e3094dfd310